package/windows/hyperkube.ps1

```diff
@@ -943,9 +943,39 @@ function init {
         try {
             $splited = $hostname -split "\."
             if ($splited.Length -lt 1) {
-                $NodeName = $splited[0]
+                $nn = $splited[0]
             }
         } catch {}
+
+        ## repair Get-GcePdName method ##
+        $getGcePodNameCommand = Get-Command -Name Get-GcePdName -ErrorAction Ignore
+        if (-not $getGcePodNameCommand) {
+            print "Can't find Get-GcePodName, try to repair ..."
+
+            $dllDownloadURL = "https://github.com/pjh/gce-tools/raw/master/GceTools/GetGcePdName/GetGcePdName.dll"
+            $dllPath = "$RancherDir\GetGcePdName.dll"
```

Comment:
Is it only for GCE? Will you try to download it if your Windows server is in your local datacenter?